keps/sig-node/3721-support-for-env-files/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Support for env files.
+kep-number: 3721
+authors:
+  - "@sharpz7"
+  - "@AnishShah"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: provisional
+creation-date: 2024-02-15
+reviewers:
+  - "@sftim"
+  - "@SergeyKanzhelev"
+approvers:
+  - "@dchen1107"
+see-also:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: implementable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.32"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.32"
```

Comment:
curious why not 1.31?

Reply:
I don't know if I will get KEP approval by 1.31 KEP freeze. I can update this if I get it.